lib/ansible/cli/galaxy.py

```diff
@@ -1379,9 +1384,11 @@ def execute_list_collection(self, artifacts_manager=None):
         :param artifacts_manager: Artifacts manager.
         """
 
+        output_format = context.CLIARGS.get('output_format', False)
```

Comment:
Since `default` + `choices` are used, it's guaranteed that the key exists and so there's no point in defaulting to `False`. There's no way it's useful. So let's not assume a missing key. If somebody refactors things and it'll start failing, then this must start failing loudly too.
```suggestion
        output_format = context.CLIARGS['output_format']
```

Reply:
Agree on this it should change.

Reply:
Done.